addons/website/static/src/client_actions/website_preview/website_preview.scss

```diff
@@ -56,6 +56,22 @@
             }
         }
     }
+
+    // Fix for Edge with 150% zoom: when entering edit mode, the page switches
+    // to mobile view because Edge adds extra border around the window, causing
+    // the width to fall below the mobile breakpoint. To prevent this, we reduce
+    // the sidebar width by 5px to keep the page in desktop view.
+    // TODO: In the next sidebar redesign, adjust the sidebar width properly to
+    // avoid this workaround.
+    &.o_is_microsoft_edge.editor_enable.editor_has_snippets {
+        @media (min-resolution: 1.5dppx) and (max-resolution: 1.51dppx) {
+            margin-right: calc(#{$o-we-sidebar-width} - 5px) !important;
```

Comment:
- Is there a reason you use `calc`? :thinking: 
- Also, instead of doing the computation twice, save it in a variable first here, then use it twice

Reply:
I don't understand what you are saying... it's defined in pixel indeed, so... good? Just `$o-we-sidebar-width - 5px` instead of `calc(#{$o-we-sidebar-width} - 5px)`? :thinking:

Reply:
I thought `calc` was needed for calculations with units like `px` :face_with_peeking_eye:
But it's not needed in SCSS, indeed. I've made the change :saluting_face: